src/sentry/static/sentry/app/views/alerts/details/activity/statusItem.tsx

```diff
@@ -50,7 +50,7 @@ class StatusItem extends React.Component<Props> {
         showTime={showTime}
         author={{
           type: activity.user ? 'user' : 'system',
-          user: activity.user,
+          user: activity.user || undefined,
```

Comment:
@markstory is this to convert `null` to `undefined`?

Reply:
Yes, the ActivityItem was complaining about getting a `null` in the optional key.